tests/test_preprocessing.py

```diff
@@ -191,6 +193,57 @@ def test_replace_x_y_nominal_lat_lon(dask, nans):
     assert set(replaced_ds.lat.dims) == set(["x", "y"])
 
 
+class TestReplaceXYNominalLatLon(TestCase):
```

Comment:
I am not really familiar with unittest. Is there a specific reason not to use pytest fixtures (or plain functions) in this case?

Reply:
Unittest comes with a few handy tools. I personally like it a lot because plain functions - in my experience - sometimes increase code duplication and add clutter for setting up some dummy data. Truthfully, I only use `TestCase.assertRaises` and  `TestCase.assertTrue` which I can easily replace with some plain code, and this specific test is really simple to do without things from Unittest. I'll change it.

Reply:
@jbusecke , the Unittest has been removed in [8f1fb69](https://github.com/jbusecke/xMIP/pull/365/commits/8f1fb6975fb0667650adfa67d1960ef0678d3a9a)